lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -647,6 +645,9 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     informed_entities
     |> Enum.map(fn %{stop: stop_id} -> stop_id end)
     |> Enum.filter(&String.starts_with?(&1, "place-"))
+    |> Enum.uniq()
+    |> Enum.map(&Stop.fetch_stop_name(&1))
```

Comment:
Noted in slack: You'll need to pass a stubbed function down into this function in order to prevent it from making API requests while being unit tested. (Also, need unit tests for this change)